.github/ISSUE_TEMPLATE/release_checklist_template.md

```diff
@@ -6,25 +6,32 @@ labels: ''
 assignees: ''
 
 ---
-Soft freeze:   
-Hard freeze: 
+Soft freeze:
+Hard freeze:
 
-Branch setup, steps typically done when creating new version branch:
+## Soft Freeze Items
+### Branch setup, steps typically done when creating new version branch:
 - [ ] Update Version Number in code (Eng task, use `update_version.sh`)
 - [ ] Update `Client/Info.plist` MozWhatsNewTopic (Eng task)
 - [ ] Create version specific Sentry project
 - [ ] Update SENTRY_DSN secret in Bitrise (per release)
-- [ ] Add Bitrise trigger for release branch 
----
+- [ ] Add Bitrise trigger for release branch
+
+### Soft Freeze Optional
+These items should be completed as soon as possible, preferrably on soft code freeze day. Due to timing issues, they may not be available until hard code freeze, and must
+
 - [ ] Ensure string export was completed for L10N (if needed)
-- [ ] Check for [security advisories](https://github.com/mozilla-mobile/firefox-ios/wiki/Release-Build-Checklist/#security-advisories) 
+- [ ] Check for [security advisories](https://github.com/mozilla-mobile/firefox-ios/wiki/Release-Build-Checklist/#security-advisories)
+- [ ] File P.I. request
+
+## Hard Code Freeze
+
 - [ ] Tag release in GitHub (Eng task)
     - [ ] Link to commit diff between versions
     - [ ] Add volunteer contributions
-- [ ] File P.I. request
 - [ ] Release Notes updated
```

Comment:
Let's make this ordering
- rel notes
- screenshots and locales
- submit
- rollout